src/components/ThemeProvider/ThemeProvider.tsx

```diff
@@ -19,7 +20,10 @@ export type ThemeProviderProps = {
 const deepMergeTheme = (newTheme: DeepPartial<Theme>, theming: 'dark' | 'semantic'): Theme =>
   merge(theme(theming), pick(newTheme, keys(theme(theming))));
 
-const ThemeProvider: React.FCC<ThemeProviderProps> = ({ theme = {}, children }) => {
+const ThemeProvider: React.FC<React.PropsWithChildren<ThemeProviderProps>> = ({
```

Comment:
this could probably be FCC

Reply:
fixed here https://github.com/Orfium/orfium-ictinus/pull/718/commits/b8d5f77f8605a2c62ea17e696ae816d9a509dc14